src/scheduler/ExecutableCronSchedule.ts

```diff
@@ -0,0 +1,48 @@
+import { parseExpression } from 'cron-parser';
+import { CronJob } from 'cron';
+import { DateTime } from 'luxon';
+import { CronSchedule } from '../job/MomoJob';
+import { ExecutableSchedule, NextExecutionTime } from './ExecutableSchedule';
+import { momoError } from '../logging/error/MomoError';
+
+export class ExecutableCronSchedule implements ExecutableSchedule<CronSchedule> {
+  private readonly cronSchedule: string;
+  private scheduledJob?: CronJob;
+
+  constructor({ cronSchedule }: CronSchedule) {
+    this.cronSchedule = cronSchedule;
+  }
+
+  toObject(): CronSchedule {
+    return { cronSchedule: this.cronSchedule };
+  }
+
+  execute(callback: () => Promise<void>): NextExecutionTime {
+    this.validateCronSchedule();
+
+    this.scheduledJob = new CronJob(this.cronSchedule, callback);
+    this.scheduledJob.start();
```

Comment:
this CronJob from the lib ... can it throw?

Reply:
Hm... shouldn't really happen but just to make sure we wrapped it in a try-catch.